app/authentication/user.py

```diff
@@ -1,8 +1,10 @@
+from typing import Optional
+
 from flask_login import UserMixin
 
 
 class User(UserMixin):
-    def __init__(self, user_id, user_ik):
+    def __init__(self, user_id: str, user_ik: Optional[str]) -> None:
```

Comment:
`user_id` is also optional as `session_store.user_id` default value is `None` and the check below also covers against it.